services/dictionary-api/src/main/java/datawave/webservice/query/result/metadata/DefaultMetadataField.java

```diff
@@ -51,9 +52,13 @@
     
     @XmlAttribute
     private Boolean normalized = false;
+    
     @XmlAttribute
     private Boolean reverseIndexed = false;
     
+    @XmlAttribute
+    private Boolean tokenized = false;
```

Comment:
You need to add support for this field in the protostuff serialization code below (see code starting at line 171/184).

Reply:
Added in the latest commit.